server/webdriver_tests/shared.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+LOADING_WAIT_TIME = 3
+"""Common library for functions used by multiple webdriver tests"""
+
+
+def wait_for_loading(driver):
+    screen_present = EC.visibility_of_element_located((By.ID, 'screen'))
+    screen_hidden = EC.invisibility_of_element_located((By.ID, 'screen'))
+    while (True):
+        try:
```

Comment:
do not quite understand the logic here? Can you add some comment?

Reply:
should we also add some max retries?

Reply:
added comments and max retries